clang/lib/CodeGen/CodeGenModule.cpp

```diff
@@ -4551,6 +4554,7 @@ llvm::Constant *CodeGenModule::GetOrCreateLLVMFunction(
 
   llvm::Function *F =
       llvm::Function::Create(FTy, llvm::Function::ExternalLinkage,
+                             getDataLayout().getProgramAddressSpace(),
```

Comment:
This doesn't do anything?  Function::Create already puts functions in the program address-space by default.